### DISCUSSION TITLE
微调后进行模型部署，网页版对话框能打开，进行对话无输出。

### DISCUSSION THREAD

**Discussion by yimisiyang** [zai-org/ChatGLM3#1049] [Q&A] [answered]
我使用的是官方ptuning demo进行训练、推理、部署，训练，推理可以跑通，运行 `bash web_demo.sh`  部署后，模型运行正常，GPU显存占用正常，但是web页面，输入问题后，无输出结果。截图如下
web_demo.sh运行图
![捕获](https://github.com/THUDM/ChatGLM3/assets/40728232/17e8aafa-7a19-4c57-b330-546d139da7c0)
web端输入问题后运行图
![捕获1](https://github.com/THUDM/ChatGLM3/assets/40728232/c238da56-9230-4678-b99a-1a804d4c4428)
GPU占用情况图，是否说明模型已成功运行？
训练推理后输出的文件图
![捕获3](https://github.com/THUDM/ChatGLM3/assets/40728232/6a4a2394-5bfd-40fb-8755-658f6abef312)
我调用的是cheakpoint 3000中的模型
![Uploading 捕获4.PNG…]()

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
修改推理代码，记得要移植infrence 中读入模型的代码到其他demo
